test/torchaudio_unittest/transforms/autograd_test_impl.py

```diff
@@ -0,0 +1,45 @@
+from parameterized import parameterized
+from torch.autograd import gradcheck, gradgradcheck
+import torchaudio.transforms as T
+
+from torchaudio_unittest.common_utils import (
+    TestBaseMixin,
+    get_whitenoise,
+)
+
+
+class AutogradTestCase(TestBaseMixin):
+    def assert_grad(self, transform, *inputs, eps=1e-06, atol=1e-05, rtol=0.001):
+        transform = transform.to(self.device, self.dtype)
+
+        inputs_ = []
+        for i in inputs:
+            i.requires_grad = True
+            inputs_.append(i.to(dtype=self.dtype, device=self.device))
+        assert gradcheck(transform, inputs_, eps=eps, atol=atol, rtol=rtol)
+        assert gradgradcheck(transform, inputs_, eps=eps, atol=atol, rtol=rtol)
```

Comment:
you might also wanna test the non contiguous inputs here (`gen_non_contig_grad_outputs=True`)

Reply:
synced with @albanD offline: this is not needed

Reply:
I don't think it is needed here.